config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  get '/new',to: 'users#new'
+  get '/users/:id/edit', to: 'users#edit'
+  patch '/users/:id/edit', to: 'users#update'
+  resources :users, :only => [:new, :create, :edit, :update]
```

Comment:
You're repeating routes here. Please only use `resources` as indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route1.png) and [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png)